### PR TITLE
Removed 250 and 1000 items/page options and set 10 as default value

### PR DESCRIPTION
**Pull request by jeffmorin**
## Description
As requested by @tdonohue, I removed the 250 and 1000 items/page options from the Filtered Items report query interface and set 10 items/page as the combobox default value.

## Instructions for Reviewers
Access the Filtered Items (aka Metadata Query) content report and check that the default Page Limit value is set to 10 and the only available options are 10, 25, 50, and 100.

List of changes in this PR:
* Removed 250 and 1000 items/page options from the `pageLimit` combobox.
* Set 10 as default value in that same combobox.

This change avoids performance losses due to too large quantities of data to be loaded in a single query.

## Checklist
_This checklist provides a reminder of what we are going to look for when reviewing your PR. You need not complete this checklist prior to creating your PR (draft PRs are always welcome). If you are unsure about an item in the checklist, don't hesitate to ask. We're here to help!_

The unchecked boxes below are not applicable to this PR.

- [x] My PR is small in size (e.g. less than 1,000 lines of code, not including comments & specs/tests), or I have provided reasons as to why that's not possible.
- [x] My PR passes [ESLint](https://eslint.org/) validation using `yarn lint`
- [x] My PR doesn't introduce circular dependencies (verified via `yarn check-circ-deps`)
- [ ] My PR includes [TypeDoc](https://typedoc.org/) comments for _all new (or modified) public methods and classes_. It also includes TypeDoc for large or complex private methods.
- [x] My PR passes all specs/tests and includes new/updated specs or tests based on the [Code Testing Guide](https://wiki.lyrasis.org/display/DSPACE/Code+Testing+Guide).
- [ ] If my PR includes new libraries/dependencies (in `package.json`), I've made sure their licenses align with the [DSpace BSD License](https://github.com/DSpace/DSpace/blob/main/LICENSE) based on the [Licensing of Contributions](https://wiki.lyrasis.org/display/DSPACE/Code+Contribution+Guidelines#CodeContributionGuidelines-LicensingofContributions) documentation.
- [ ] If my PR includes new features or configurations, I've provided basic technical documentation in the PR itself.
- [ ] If my PR fixes an issue ticket, I've [linked them together](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

### CODE REVIEW

File: src/app/admin/admin-reports/filtered-items/filtered-items.component.ts [hunk 1]
```diff
@@ -293,6 +291,14 @@ export class FilteredItemsComponent {
       );
   }
 
+  /*postFilteredItems(): Observable<RawRestResponse> {
```
Comment (tdonohue) [inline src/app/admin/admin-reports/filtered-items/filtered-items.component.ts:294]: @jeffmorin : Could we remove this commented out code?  It appears to have be added accidently?

**Review by tdonohue — COMMENTED** [review]
Thanks @jeffmorin !  This looks good overall & works, but it has some commented out code that should be included?  It appears to have been added accidently.  Once that code is removed this looks good to merge.

File: src/app/admin/admin-reports/filtered-items/filtered-items.component.ts (hunk 1 above)
Comment (jeffmorin) [reply src/app/admin/admin-reports/filtered-items/filtered-items.component.ts]: Done. Sorry about that!

**Review by tdonohue — APPROVED** [review]
👍 Thanks @jeffmorin !  Looks good now!

**Pull request merged**